CMakeLists.txt

```diff
@@ -502,8 +502,10 @@ function(
 
     if(target_triple MATCHES "^aarch64")
         set(cpu_family aarch64)
+        set(enable_long_double_test false)
```

Comment:
If the reason for failure of long double tests is known, it would be worth putting it in a commend here.